pkg/semver/version.go

```diff
@@ -23,6 +23,8 @@ import (
 var (
 	ScyllaVersionThatSupportsArgs              = semver.MustParse("4.2.0")
 	ScyllaVersionThatSupportsDisablingIOTuning = semver.MustParse("4.3.0")
+	//TODO: verify that
```

Comment:
You can double check, but i validated it that starting from 2021, perftune has GKE support

Reply:
how does this semver compare works across OSS and Enterprise?

Reply:
None of released OSS version have support for disabling writeback cache. Current master has it, so probably 4.6 is going to support it. 
4.x.x is smaller than 2021.x.x so extra argument won't be added on gke nodes.

Reply:
but you wan't be able to change it when 4.6 comes, right?

Reply:
Yes, we will have to update operator. So far we don't know if it will be included in 4.6, so we cannot set anything in stone.

Reply:
but combining enterprise versions here like this preclude it from ever working by design. We should compare OSS and Enterprise versions separately.

Reply:
now i see what you mean, yes we need to compare version separately